Dockerfile

```diff
@@ -21,6 +35,8 @@ PKGS_TO_INSTALL=$(cat <<-END\n\
   python36u-devel\n\
   which\n\
   wget\n\
+  mariadb-devel\n\
+  sqlite\n\
```

Comment:
It is usually a good practice is to keep such lists alphabetically sorted to avoid duplicate inclusion in the future as the list grows.

Reply:
Good catch. Fixed in 4e47107